tests/unit/Error.spec.js

```diff
@@ -0,0 +1,16 @@
+import { mount } from "@vue/test-utils";
+import Error from "@/components/common/Error.vue";
+
+test("Error works", () => {
+  const errorComponent = mount(Error, {
```

Comment:
jeżeli nie potrzebujesz montować komponentów używanych wewnątrz tego komponentu to używaj shallowMount zamiast mount- szybciej działa bo pomija proces montowania komponentów zagnieżdżonych

Reply:
zamiast errorComponent użyj nazwy wrapper, taka jest konwencja